addon/Config.jsm

```diff
@@ -37,31 +37,7 @@ var config = {
      * - If there is no key for an endStudy reason, no url will open.
      * - usually surveys, orientations, explanations
      */
-    endings: {
```

Comment:
Since we're running the experiment in stealth-mode, we don't need to open tabs for specific endings.
We only don't experiment with multiple variations; it's ON or OFF.